app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/campaign/Viewmodel/ActiveCampaignViewModel.kt

```diff
@@ -0,0 +1,61 @@
+package org.wikiedufoundation.wikiedudashboard.ui.campaign.Viewmodel
+
+import androidx.lifecycle.*
+import kotlinx.coroutines.launch
+import org.wikiedufoundation.wikiedudashboard.ui.campaign.Repository.ActiveCampaignRepository
+import timber.log.Timber
+
+/**
+ * Class extends AndroidViewModel and requires application as a parameter.
+ */
+
+class ActiveCampaignViewModel(private val activeCampaignRepository: ActiveCampaignRepository) : ViewModel() {
```

Comment:
Change package Name ViewModel -> viewmodel.

package-name should not contain camelCase or snake_case letters.

https://stackoverflow.com/a/3179277